packages/sdk/src/Incentives/AllowListIncentive.ts

```diff
@@ -272,7 +272,7 @@ export class AllowListIncentive extends DeployableTarget<
   ) {
     return await readAllowListIncentiveIsClaimable(this._config, {
       address: this.assertValidAddress(),
-      args: [prepareClaimPayload(payload)],
+      args: [payload.target, zeroHash],
```

Comment:
It's always cheaper to pass in an empty bytestring when the parameter is unused
```suggestion
      args: [payload.target, "0x"],
```